CMP/CmpWap/CmpWapExtension/Api/Models/MicrosoftMgmtSvcCmpContext.cs

```diff
@@ -24,9 +24,12 @@ public MicrosoftMgmtSvcCmpContext()
 
         public static string GetConnectionString()
         {
-            var xk = new KryptoLib.X509Krypto();
-            return xk.GetKTextConnectionString("MicrosoftMgmtSvcCmpContext",
-                "MicrosoftMgmtSvcCmpContextPassword");
+            using (var xk = new KryptoLib.X509Krypto())

```

Comment:
Fantastic inclusion. Yes, when using krypto, we can use each instance only once, so this is a little more "safe".